src/main/java/sirius/db/mixing/properties/AmountProperty.java

```diff
@@ -111,14 +114,21 @@ protected Object transformValueFromImport(Value value) {
 
     @Override
     protected Object transformToJDBC(Object object) {
-        return object == null || ((Amount) object).isEmpty() ? null : ((Amount) object).getAmount();
+        Amount amount = (Amount) object;
+        if (amount == null || amount.isEmpty()) {
+            return null;
+        }
+        NumberFormat format = getAnnotation(Numeric.class).map(numeric -> {
+            return new NumberFormat(numeric.scale(), RoundingMode.HALF_UP, NLS.getMachineFormatSymbols(), false, null);
+        }).orElse(NumberFormat.MACHINE_THREE_DECIMAL_PLACES);
+        return amount.toString(format);
```

Comment:
why would we choose a string over a BigDecimal? many JDBC drivers (esp. IBM i5 expect a BigDecimal here).

Also, rounding within a coercion method sounds like a risky idea